Formula/bazelisk.rb

```diff
@@ -15,11 +16,27 @@ class Bazelisk < Formula
 
   depends_on "bazel" => :build
 
+  conflicts_with "bazel", :because => "Bazelisk replaces the bazel binary"
```

Comment:
It can't depend on and conflict with Bazel. If you want to install a `bazel` binary it'll need to be keg-only.